front/scripts/send_tracker_generations.ts

```diff
@@ -0,0 +1,101 @@
+import { sendTrackerWithGenerationEmail } from "@app/lib/api/tracker";
+import { TrackerGenerationModel } from "@app/lib/models/doc_tracker";
+import { frontSequelize } from "@app/lib/resources/storage";
+import { DataSourceModel } from "@app/lib/resources/storage/models/data_source";
+import { isEmailValid } from "@app/lib/utils";
+import { makeScript } from "@app/scripts/helpers";
+
+makeScript(
```

Comment:
NIT: we had a cli specific for trackers, can we either kill the cli or kill that and put it in the cli to have one single system to debug?
(was in front/temporal/tracker/admin/cli)